src/ng/directive/input.js

```diff
@@ -879,9 +905,18 @@ var inputType = {
              }]);
          </script>
          <form name="myForm" ng-controller="ExampleController">
-           <input type="radio" ng-model="color.name" value="red">  Red <br/>
-           <input type="radio" ng-model="color.name" ng-value="specialValue"> Green <br/>
-           <input type="radio" ng-model="color.name" value="blue"> Blue <br/>
+           <label>
+             <input id="red" type="radio" ng-model="color.name" value="red">
+             Red
+           </label><br/>
+           <label>
+             <input id="green" type="radio" ng-model="color.name" ng-value="specialValue">
+             Green
+           </label><br/>
+           <label>
+             <input id="blue" type="radio" ng-model="color.name" value="blue">
+             Blue
+           </label><br/>
```

Comment:
Good question, that was leftover from my first pass on the code. They are no longer necessary!